code/tutorial_code/tutorials_04/hiring/check_availability.sl

```diff
@@ -0,0 +1,21 @@
+namespace: tutorials_04.hiring
+
+operation:
+  name: check_availability
+
+  inputs:
+    - address
+
+  action:
+    python_script: |
+      import random
+      rand = random.randint(0, 2)
+      vacant = rand != 0
+      #print rand
```

Comment:
Intended. 

From the lesson:

"We’ve added a commented-out line, using a Python comment (#) to print the random number that was generated. We can uncomment this line during testing to see that our operation is working as expected."